src/app/features/search/multiple-search-results/multiple-search-results.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Location } from '@angular/common';
+import { Component, OnDestroy } from '@angular/core';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
+import { select, Store } from '@ngrx/store';
+import { SEARCH_TYPES, TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { selectQueryParams } from '@store/router/selectors/router.selectors';
+import { Observable, Subject, takeUntil } from 'rxjs';
+
+@Component({
+  selector: 'app-multiple-search-results',
+  templateUrl: './multiple-search-results.component.html',
+  styleUrls: ['multiple-search-results.component.scss']
+})
+export class MultipleSearchResultsComponent implements OnDestroy {
+  vehicleTechRecords$: Observable<VehicleTechRecordModel[]>;
+  ngDestroy$ = new Subject();
+
+  constructor(public globalErrorService: GlobalErrorService, private technicalRecordService: TechnicalRecordService, private store: Store, private location: Location) {
+    this.store //
```

Comment:
// ?

Reply:
I have Prettier messing up my code on saving and I put this to disable that. It's sorted now. This is removed.